README.md

```diff
@@ -64,44 +68,46 @@ Example:
+### 2. Nodes with transform-style: `flat` prevent descendents from being detached.
 
-[//]: # (TODO: We need to explan this, but I cant think of the best explanation)
+Transform style `flat` and effects that cause flattening ([spec](https://drafts.csswg.org/css-transforms-2/#grouping-property-values)) will cause `detached` fall back to `preserve-3d` behavior in descendent elements.
 
+The one exception is root node which allows detached descendants even though an implicit `flat` transform-style is applied.
 
-### 3. Effects such as clip applied to parents of detached nodes are not applied to detached nodes.
+### 3. Only the top-most frame can use this feature.
 
-The detached elements and their descendants are rendered onto surfaces that are distinct from their parents. Any effect applied to parents will be applied during the compositing stage and will not be applied to the detached elements.
+iframes should not be allowed to created detached content.
+This feature will only be available to the main frame.
+In iframes, `detached` will fall back to `preserve-3d`.
 
-Effects that cause flattening (ie filters, opacity) will cause the element to revert to `flat` behavior. 
+### 4. Detached surfaces will not participate in stacking contexts.
 
-### 4. Only the top-most frame can use this feature.
-iframes should not be allowed to created detached content. This feature will only be available to the main frame. In iframes, `detached` will be treated at `preserve-3d`.
+Detached surfaces will be allowed to intersect with and pass through elements in the parent (and ancestor) stacking contexts.
 
-## Detailed design discussion
+Current browsers exhibit conflicting behavior in this regard - Rotated elements pass through sibling elements from the same stacking context in Safari, while in Chromium, they do not.
 
-[//]: # ([Tricky design choice #1])
-[//]: # ([Talk through the tradeoffs in coming to the specific design point you want to make.])
+### 5. When detached elements are nested, only the top most element will signal that its children at detached
 
-[//]: # (// Illustrated with example code.)
-[//]: # ([This may be an open question, in which case you should link to any active discussion threads.])
+Allowing detached descendants of detached nodes significantly complicates the implementation and introduces multiple corner cases.
+We propose that descendents with `detached` are treated as `preserve-3d`.
 
-[//]: # ([Tricky design choice 2])
-[//]: # ([etc.])
-
-## Considered alternatives
+[//]: # (TODO: We need to explain this, but I cant think of the best explanation)
 
+#
+## Considered alternatives / other details
 [//]: # ([This should include as many alternatives as you can, from high level architectural decisions down to alternative naming choices.])
 
+### 1. Applying the detached style directly to the surface to be detached.
```

Comment:
Really can't figure out the difference between Alternative 1 and 2. Need a more cleaner flow of explainer.

Reply:
the difference is not between alternative 1 and 2. The difference is between main proposal vs. alternative 1 and main proposal vs. alternative 2

Reply:
title changed